src/Apps/Auction/Routes/Bid/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Separator, Serif } from "@artsy/palette"
+import { BidFormFragmentContainer as BidForm } from "Apps/Auction/Components/BidForm"
+import { LotInfoFragmentContainer as LotInfo } from "Apps/Auction/Components/LotInfo"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { trackPageViewWrapper } from "Apps/Order/Utils/trackPageViewWrapper"
+import { track } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import React from "react"
+import { Title } from "react-head"
+import { createFragmentContainer, RelayProp } from "react-relay"
+import { TrackingProp } from "react-tracking"
+import createLogger from "Utils/logger"
+
+const logger = createLogger("Apps/Auction/Routes/Bid")
+
+interface BidProps {
+  artwork: any
+  me: any
+  relay: RelayProp
+  tracking: TrackingProp
```

Comment:
We can probably remove everything related to tracking for now, including the schema change. We can circle back to that when we're ready to add all of the desired events.